### ISSUE TITLE
The English fallback for untranslated strings in other UI languages is not inflected for singular/plural

### ISSUE THREAD

**Issue by Yorwba** [Tatoeba/tatoeba2#2336]
Using the Chinese UI and the new sentence design, [Sentence #1](https://tatoeba.org/cmn/sentences/show/1) looks like this:
![show 1 more translation](https://user-images.githubusercontent.com/6555947/82158971-8f48a400-988b-11ea-93b9-ad219aedfa04.png)
Note that the text at the bottom says "show 1 more translation". But actually there are many more translations, and this is displayed correctly in the English UI:
![show 312 more translations](https://user-images.githubusercontent.com/6555947/82159022-cd45c800-988b-11ea-8857-d22be77766cf.png)
I thought that maybe someone made a mistake translating this string, but it was actually untranslated.
After I added a translation, the number is now correct on dev:
![显示其他的299个翻译](https://user-images.githubusercontent.com/6555947/82159066-13029080-988c-11ea-992d-270da8c794d3.png)

So this must be a problem with the fallback selection.

**Comment by Yorwba** [comment]
Addendum: I tried a few of the other UI languages, and there are a few where the string is untranslated, but displayed correctly (e.g. Azerbaijani, Frisian) and others where it shows the wrong number (e.g. Malay). I suspect this depends on whether the language in question has a singular/plural distinction as well or not.

**Comment by jiru** [comment]
Yes, I’ve seen that bug too. I think the Chinese plural rule is applied as-is to the English sentence.
